impeller/renderer/backend/vulkan/driver_info_vk.cc

```diff
@@ -336,22 +336,18 @@ bool DriverInfoVK::IsEmulator() const {
 
 bool DriverInfoVK::IsKnownBadDriver() const {
   if (adreno_gpu_.has_value()) {
-    auto adreno = adreno_gpu_.value();
-    switch (adreno) {
-      // See:
-      // https://github.com/flutter/flutter/issues/154103
-      //
-      // Reports "VK_INCOMPLETE" when compiling certain entity shader with
-      // vkCreateGraphicsPipelines, which is not a valid return status.
-      // See https://github.com/flutter/flutter/issues/155185 .
-      case AdrenoGPU::kAdreno630:
-      // See:
-      // https://github.com/flutter/flutter/issues/155185
-      // Unknown crashes but device is not easily acquirable.
-      case AdrenoGPU::kAdreno506:
-        return true;
-      default:
-        return false;
+    AdrenoGPU adreno = adreno_gpu_.value();
+    // See:
```

Comment:
Should this comment block also link to https://github.com/flutter/flutter/issues/159834 ?

Reply:
same error basically, no additional info.